addons/api/addon/utils/sqlite-query.js

```diff
@@ -75,6 +81,28 @@ function addFilterConditions({ filters, parameters, conditions }) {
       continue;
     }
 
+    const firstOperator = Object.keys(filterValueArray[0])[0];
+    const allOperatorsEqual = filterValueArray
+      .flatMap((item) => Object.keys(item))
+      .every((op) => op === firstOperator);
+    if (
+      filterValueArray.length > MAX_EXPR_NUM &&
```

Comment:
Ended up adding the limit back in to reduce the number of filters that will need to use IN/NOT IN operators since this part doesn't make use of '?' for parameters.

Reply:
Why can't we parametrize it? It should be possible like in our delete statements
https://github.com/hashicorp/boundary-ui/blob/4c7ea161cd6981530cd320f1b19f3b559a47e299/addons/api/addon/workers/utils/schema.js#L474